client/app/components/ServicesTemplate/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import cn from 'classnames';
+import ServicesCardContent from './ServicesCardContent';
+import Card from 'components/SharedComponents/Card';
+import json from './data.json';
+
+// Styles
+import styles from './styles.scss';
+
+const ServicesTemplate = () => {
+  const filledCardsTop = json.data.topCardContent.map(({ title, subtitle, content, url, img, id }) =>
+    <Card
+      key={id}
+      cardSize = "is-6"
```

Comment:
Are you sure AirBnB linter rules are working on your setup? Stuff like spaces around equals in assigning props or not having spaces between object keys should be popping up.
![screen shot 2018-01-05 at 8 25 39 am](https://user-images.githubusercontent.com/10067791/34617971-0f2e9eb0-f1f2-11e7-943c-26f31c97698c.png)